pagopa/io-web-profile
# [IOPID-1233] : Error Login Page Buttons

#  ⚠️ Depends on https://github.com/pagopa/io-web-profile/pull/120⚠️

## Short description

The purpose of this PR is to fix the behavior of the buttons on the login error page, as described in the[ JIRA Task.](https://pagopa.atlassian.net/browse/IOPID-1233)


## How to test

To test this, start the project and attempt a login. On the hubspidlogin page, click "Cancel" without completing the login; you will be redirected to the login error page.

Ensure that the two CTA "Close" and "Retry" on the page behave as follows:

Close: should return the user to the previous screen with the choice of SPID or CIE IdP (which is not always the portal access screen at https://ioapp.it/it/accedi/). For instance if you was trying to logout from it/esci/ you should be redirect to it/esci/

Retry: should reopen the IdP access screen.
